### ISSUE TITLE
Using interface=ib0 interferes with serving bokeh

### ISSUE THREAD

**Issue by caseyjlaw** [dask/distributed#1687]
(After asking around with my sysadmins, I think this is not just related to my network configuration. But I could be wrong.)

I am using infiniband (--interface=ib0 for dask-scheduler and dask-worker) on a small linux cluster. The only change I needed was to refer to the host by its IP rather than IP, as identified by at scheduler startup. After that, I can see all processes start well and seem to run tasks, etc..

However, after this change I cannot find the bokeh web server. The scheduler reports an IP address and port for the bokeh server. The IP address is the same as that of the scheduler and the port is 1 higher, as usual. But using my usual ssh tunneling to that address:port does not expose the bokeh server. 

My sysadmins noted that the infiniband is configured to handle TCP/IP traffic, but that is only intended to communicate with other devices on infiniband. Outside traffic is not supported. Their suggestion is to explcitly tell the web server the device to listen to. Is this supported by an existing scheduler option?

**Comment by mrocklin** [comment]
I agree with your sysadmins and think that this *should* be supported.  Unfortunately it currently is not.  

The offending line is here: https://github.com/dask/distributed/blob/7b987c2ba29ba2fe0acc7f7dd087d351ee076129/distributed/scheduler.py#L985

We should be able to pass in that `listen_ip` address from the command line.  This will require snaking it in through a few function calls.  This will require changing the `Scheduler(services=)` keyword in such a way that provides adding this option and is also backwards compatible.

If you have *any* interest in taking this on that would be quite welcome.

**Comment by rabernat** [comment]
I encountered this same error on my local cluster. I worked around it by *not* using infiniband for the scheduler but continuing to use it for the workers.

**Comment by caseyjlaw** [comment]
@rabernat Thanks for that. I don't have a lot of data moving to my scheduler, so that works for me.
I looked over the changes, but don't feel comfortable enough with a PR. I'll go ahead without ib0 on the scheduler.

**Comment by caseyjlaw** [comment]
@rabernat I definitely feel encouraged to submit a PR. After reviewing the code, I don't think I understand the theory well enough to do so.
